### PR TITLE
Adding changes in Release Notes as per CP-6148

### PR DESCRIPTION
**Pull request by Mona2012**

### CODE REVIEW

File: pages/04.HDM Doc 2.2/15.release-notes/default.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ What is new in HDM 2.2.1 Release?
 *   Provides support for Cold, TBC and Warm Migration to VMware Cloud Director.
 *   Highly efficient, proprietary, data transport for Cold, TBC and Warm migration.
 *   WAN disconnects lasting more than 2 minutes will be recovered.
+*   Static IP support is provided during migration on VC-VCD.
```
Comment (kgcb) [inline pages/04.HDM Doc 2.2/15.release-notes/default.md]: Doesn't seem like we have mentioned the short forms VC & VCD in this section before this.

Also keeping in line with the phrases used in this section earlier better to use "Static IP support is provided during migration to VMware Cloud Director."

**Review by kgcb — COMMENTED** [review]
Line numbered 5 & 7 have additional spaces before _Lite-Cluster_.

File: pages/04.HDM Doc 2.2/15.release-notes/default.md (hunk 1 above)
Comment (ayyajpio) [reply pages/04.HDM Doc 2.2/15.release-notes/default.md]: Agreed. Manpreet, please update this.

Comment (Mona2012) [reply pages/04.HDM Doc 2.2/15.release-notes/default.md]: Resolved..

**Review by ayyajpio — APPROVED** [review]
Looks good to me.

**Pull request merged**